src/components/PieChart.js

```diff
@@ -1,57 +1,49 @@
-import React, { useState } from 'react';
-import { VictoryPie, VictoryLegend } from 'victory';
+import React from 'react';
+import { VictoryPie, VictoryLegend, VictoryTooltip } from 'victory';
 import useMediaQuery from '@material-ui/core/useMediaQuery';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components/macro';
 import PropTypes from 'prop-types';
 
 import { privateColorScale, attributeColors } from '../constants/attribute-colors';
 
-const LabelContainer = styled('div')`
-  position: relative;
-  top: ${props => (props.smScreen ? '165px' : props.mdScreen ? '210px' : '165px')};
-  left: ${props => (props.smScreen ? '105px' : props.mdScreen ? '160px' : '80px')};
-  width: 140px;
-  height: 50px;
+const ChartWrapper = styled('div')`
+  width: 100%;
+  padding-top: 20px;
+  display: flex;
+  flex-direction: ${props => (props.mediaMin600_Max960 || !props.mediaMax1280 ? 'row' : 'column')};
+  filter: ${props => props.policy && 'blur(0.8rem)'};
 `;
-const PrivateLabelContainer = styled('div')`
+
+const PrivateTextWrapper = styled('div')`
   position: relative;
-  top: ${props => (props.mdScreen ? '210px' : '165px')};
-  left: ${props => (props.mdScreen ? '90px' : '150px')};
+  height: 0;
   width: 60%;
+  top: ${props => (props.mediaMin600_Max960 ? '170px' : '180px')};
+  left: ${props => (props.mediaMin960_Max1280 || props.mediaMax600 ? '100px' : '150px')};
   z-index: 1;
-`;
-
-const Label = styled('div')`
-  opacity: ${props => (props.content ? 1 : 0)};
-  transition: opacity 0.5s ease-in-out;
+  user-select: none;
   text-align: center;
-  font-size: 13px;
-  &::before {
-    content: '${props => props.content}';
-  }
-`;
-const PrivateLabel = styled('div')`
-  text-align: center;
-  font-size: 22px;
-  &::before {
-    content: '${props => props.content}';
-  }
+  font-size: large;
 `;
 
-const ChartWrapper = styled('div')`
-  width: 100%;
-  height: ${props => (props.mdScreen ? '450px' : '100%')};
-  display: flex;
-  flex-direction: ${props => (props.mdScreen ? 'column' : 'row')};
-  filter: ${props => props.policy && 'blur(0.8rem)'};
+const SvgWrapper = styled('svg')`
+  width: 360px;
+  ${props =>
+    props.setHeight &&
+    css`
+      height: 380px;
+    `}
 `;
 
 const PieChart = ({ protocolName, chartContent, setToPrivate, privateLabel }) => {
```

Comment:
This is a note to myself. When the review is ready, I should take a really strong look at this.